swarm/network/stream/stream.go

```diff
@@ -538,6 +520,7 @@ func (c *client) NextInterval() (start, end uint64, err error) {
 		return 0, 0, err
 	}
 	start, end = i.Next()
+	log.Debug("client.NextInterval", "start", start, "end", end)
```

Comment:
You might want to add this at the top in a `defer` statement, so that we also cover the case when there is an error.